package.json

```diff
@@ -43,7 +43,7 @@
     "@angular/platform-server": "^4.2.1",
     "@angular/router": "^4.2.1",
     "ajv": "^5.2.2",
-    "amo-tools-suite": "^0.1.45",
+    "amo-tools-suite": "file:../AMO-Tools-Suite",
```

Comment:
@rmroot should this be changed back the the .45 version number? I know you were using this to test because the tools-suite on the npm repo doesn't have the associated changes yet.